components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -257,16 +257,6 @@ - (CGSize)preferredSizeWithWidth:(CGFloat)width {
 
 #pragma mark UITextField Accessor Overrides
 
-- (void)setBorderStyle:(UITextBorderStyle)borderStyle {
```

Comment:
what is calling this now? it looks like this is marked NS_UNAVAILABLE in the header. I dont understand how this is compiling:  the subclass is calling it's super class's invocation even though it shouldn't be allowed to.

Reply:
Hmmm wait where is it marked `NS_UNAVAILABLE`?

Reply:
oh this might be marked so in the older text field:
https://github.com/material-components/material-components-ios/blob/develop/components/TextFields/src/MDCTextField.h#L34

maybe we should do the same?

Reply:
haha yea its the old one. sorry about that. I was really confused as to why it was compiling.

IMO I think we should also mark it unavailable in the newer one. I like compile time check more than runtime ones :)

Reply:
Ok 👍 . I'll mark it as unavailable on the filled and outlined textfield, but not the base one. borderSttyle doesn't conflict the base one.

Reply:
Marked as unavailable 👍